homeassistant/components/reolink/select.py

```diff
@@ -67,6 +67,17 @@ class ReolinkSelectEntityDescription(
         supported=lambda api, ch: api.supported(ch, "ptz_presets"),
         method=lambda api, ch, name: api.set_ptz_command(ch, preset=name),
     ),
+    ReolinkSelectEntityDescription(
```

Comment:
Should a translation be provided? (For the `off` case?)

Reply:
The "off" is hard coded text in the reolink-aio library
The 3 messages shown in the image are provided by the reolink doorbell itself (in the firmware) and can not be removed/renamed.
Up to 12 aditional messages can be recorded by the user and given a name in the reolink phone app.

I will add a translations for the "off" case.
I do not think we should translate the "Hi, please leave the packege at the door. We will get it later." since the spoken audio message will also always be in english (build in the firmware, can not be changed).

Reply:
@frenck translations are now provided